net/dns/publicdns/publicdns.go

```diff
@@ -23,6 +26,9 @@ var dohOfIP = map[netip.Addr]string{} // 8.8.8.8 => "https://..."
 var dohIPsOfBase = map[string][]netip.Addr{}
 var populateOnce sync.Once
 
+const NextDNSBase = "https://dns.nextdns.io/"
+const ControlDBase = "https://dns.controld.com/"
```

Comment:
if these need to be exported, they need comments.

Reply:
added comments